templates/checkout/payment.php

```diff
@@ -10,7 +10,7 @@
  * happen. When this occurs the version of the template file will be bumped and
  * the readme will list any important changes.
  *
- * @see https://docs.woocommerce.com/document/template-structure/
+ * @see     https://docs.woocommerce.com/document/template-structure/
  * @package WooCommerce/Templates
  * @version 3.4.0
```

Comment:
Can you please bump this version number to 3.5.3 for inclusion in the next fix release?

Reply:
Done.